config/locales/pt-BR.yml

```diff
@@ -66,11 +66,11 @@ pt-BR:
       destroy:
         success:
           title: "Sucesso!"
-          message: "O %{model_name} foi deletado com sucesso."
+          message: "O registro de %{model_name} foi deletado com sucesso."
 
         failure:
           title: "Atenção!"
-          message: "Falha ao excluir %{model_name}."
+          message: "O registro de %{model_name} foi deletado com sucesso."
```

Comment:
Should this line be different? This should indicate that a record could not be deleted, but looks to be the same as the successful deletion.

Reply:
You are correct, this is a wrong translation that says the item was SUCCESSFULLY deleted.